### PR TITLE
fix(query): drop table unknown table should return error

### LINKED ISSUE #10425: bug: drop not exists table success
### Search before asking

- [X] I had searched in the [issues](https://github.com/datafuselabs/databend/issues) and found no similar issues.

### Version

```sql
mysql> select version();
+-----------------------------------------------------------------------------------+
| version()                                                                         |
+-----------------------------------------------------------------------------------+
| 8.0.26-v0.9.54-nightly-c4e3dc6(rust-1.68.0-nightly-2023-03-08T06:00:33.14241861Z) |
+-----------------------------------------------------------------------------------+
1 row in set (0.01 sec)
```

### What's Wrong?

as titile.

### How to Reproduce?

```sql
mysql> show tables;
+-------------------+
| tables_in_default |
+-------------------+
| hits_parquet      |
| tc                |
| test              |
| tt                |
+-------------------+
4 rows in set (0.12 sec)
Read 37 rows, 5.71 KiB in 0.100 sec., 370.64 rows/sec., 57.23 KiB/sec.

mysql> drop table ttttt;
Query OK, 0 rows affected (0.06 sec)

```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by TCeason**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #10425

### CODE REVIEW

**Comment by BohuTANG** [comment]
Should add logic test for it.

**Comment by TCeason** [comment]
> Should add logic test for it.

Ok, Let's wait for the existing tests to succeed.

File: tests/sqllogictests/suites/base/05_ddl/05_0000_ddl_drop_tables [hunk 1]
```diff
@@ -13,6 +13,6 @@ DROP TABLE IF EXISTS t
 statement error 1025
 DROP TABLE t
 
-statement error 1002
+statement error 1025
 DROP TABLE system.abc
```
Comment (BohuTANG) [inline tests/sqllogictests/suites/base/05_ddl/05_0000_ddl_drop_tables:17]: It would be better to add:
DROP TABLE IF EXISTS system.abc

Comment (TCeason) [reply tests/sqllogictests/suites/base/05_ddl/05_0000_ddl_drop_tables:17]: Add test for 

drop table/view; 

alter table xx drop column

**Review by andylokandy — APPROVED** [review]

**Pull request merged**